platform/linux-generic/include/odp_global_data.h

```diff
@@ -61,6 +61,15 @@ typedef struct odp_global_data_ro_t {
 	config_t libconfig_runtime;
 	odp_random_kind_t ipsec_rand_kind;
 
+	/* Disabled features during global init */
+	struct {
+		uint8_t compress;
+		uint8_t crypto;
+		uint8_t ipsec;
+		uint8_t traffic_mng;
```

Comment:
Perhaps tm or traffic_mngr would be more consistent with the current internal naming.

Reply:
OK